### ISSUE TITLE
Loading on empty List

### ISSUE THREAD

**Issue by lightglitch** [naver/android-pull-to-refresh#7]
When the list is empty and we force a manual refresh the loading animation stays hidden.

**Comment by ncoolz** [comment]
I think it is a good idea to refresh automatically when list is empty. I will add a feature of it.

**Comment by lightglitch** [comment]
But the issue is not  that, the problem is that the animation does not show.

**Comment by ncoolz** [comment]
It doesn't happen to me. Can you please give me a sample code?

**Comment by lightglitch** [comment]
Sure, I will try to make it today.

**Comment by ncoolz** [comment]
Thank you.

**Comment by lightglitch** [comment]
Here you have it:

https://docs.google.com/uc?id=0BzWUSuVCNkKlMGY4RFhZd1AxWkk&export=download

If you you add elements when initializing the adapter it show's the loading animation correctly, but if don't it's hidden.

**Comment by ncoolz** [comment]
I checked the issue on your code. A cause of the issue is from below code. 

``` java
public class PullToRefreshListView extends PullToRefreshAdapterViewBase<ListView> {
...
    @Override
    protected void onRefreshing(final boolean doScroll) {
        /**
         * If we're not showing the Refreshing view, or the list is empty, the
         * the header/footer views won't show so we use the normal method.
         */
        ListAdapter adapter = mRefreshableView.getAdapter();
        if (!mListViewExtrasEnabled || !getShowViewWhileRefreshing() || null == adapter || adapter.isEmpty() || getCurrentMode() == Mode.GOOGLE_STYLE) {
            super.onRefreshing(doScroll);
            return;
        }
...
}

```

It skips a code to show a loading animation on refreshing If a list is empty as above. This issue will be fixed in next version.

**Comment by lightglitch** [comment]
Thx

**Comment by lightglitch** [comment]
It's possible to do a quick release? It would be very useful and I would be very appreciated.

**Comment by ncoolz** [comment]
Yes, It may be possible.

**Comment by ncoolz** [comment]
resolved in 3.2.1
